code/utils/include/allscale/utils/serializer.h

```diff
@@ -389,6 +463,36 @@ namespace utils {
 	deserialize(Archive& a) {
 		return ArchiveReader(a).read<T>();
 	}
+#endif
 
 } // end namespace utils
 } // end namespace allscale
+
+namespace hpx {
+namespace serialization {
+    template <typename T>
+    typename std::enable_if<
+        ::allscale::utils::is_serializable<T>::value &&
+        !(std::is_integral<T>::value || std::is_floating_point<T>::value),
+        output_archive&
```

Comment:
output_archive is not defined here, if the ALLSCALE_WITH_HPX macro is not defined. So I guess this block also needs to be enclosed by a guard

Reply:
Good catch. fixed that one.